doc/source/conf.py

```diff
@@ -102,6 +102,7 @@
     r"https://github.com/ansys-internal/.*",
     r"https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/*",
     "https://opensource.org/blog/license/mit",  # 403 - protected from bots
+    "https://github.com/ansys/{repo-name}/releases/tag/v{tag}",  # Sample URL in doc-deploy-changelog
```

Comment:
```suggestion
    r"https://github.com/ansys/.*",
```
Since link check can't work on Github links and other ansys links might be added later, let's just ignore them all.

Reply:
+1